upgrade_test.py

```diff
@@ -46,7 +48,14 @@ def truncate_entries(func):
     def inner(self, *args, **kwargs):
         node = args[0]
         with self.db_cluster.cql_connection_patient(node, keyspace='truncate_ks') as session:
-            self.cql_truncate_simple_tables(session=session, rows=NUMBER_OF_ROWS_FOR_TRUNCATE_TEST)
+            InfoEvent(message="Start truncate simple tables").publish()
+            try:
+                self.cql_truncate_simple_tables(session=session, rows=NUMBER_OF_ROWS_FOR_TRUNCATE_TEST)
+            except Exception as details:  # pylint: disable=broad-except
```

Comment:
out of curiosity, a timeout will raise an exception too, right?

Reply:
I added all exception details to the error event.
We do not want to raise Exception as it will stop the test. In this case it's not a goal. The test will fail as got error event, but will continue.

Reply:
yes, i understand that we won't raise the exception, but my question was, a timeout (inside the try clause) will raise an exception, right?

Reply:
> yes, i understand that we won't raise the exception, but my question was, a timeout (inside the try clause) will raise an exception, right?

Ah, I see. Yes, it will